pcsx2/gui/AppMain.cpp

```diff
@@ -236,8 +291,9 @@ void Pcsx2App::PadKeyDispatch( const keyEvent& ev )
 	m_kevt.SetEventType( ( ev.evt == KEYPRESS ) ? wxEVT_KEY_DOWN : wxEVT_KEY_UP );
 	const bool isDown = (ev.evt == KEYPRESS);
 
+//returns 0 for normal keys and a WXK_* value for special keys
 #ifdef __WXMSW__
-	const int vkey = wxCharCodeMSWToWX( ev.key );	//returns 0 if plain ascii value or a WXK_... (<=32 or >=300) if a special key
+	const int vkey = TranslateVKToWXK(ev.key);
```

Comment:
Well, it seems to me that for linux it uses `TranslateGDKtoWXK` anyway, and your new function `TranslateVKToWXK` only translates `VK_` which AFAIK is windows only... am I missing something?